tests/Internal/Codebase/InternalCallMapHandlerTest.php

```diff
@@ -378,31 +365,17 @@ class InternalCallMapHandlerTest extends TestCase
         'pdo::sqlitecreateaggregate',
         'pdo::sqlitecreatecollation',
         'pdo::sqlitecreatefunction',
-        'pdostatement::__sleep',
-        'pdostatement::__wakeup',
         'simplexmlelement::__get',
         'simplexmlelement::offsetexists',
         'simplexmlelement::offsetget',
         'simplexmlelement::offsetset',
         'simplexmlelement::offsetunset',
```

Comment:
SimpleXmlElement implements array access even though it's not documented. If we keep offset functions will that help psalm accept array access on the object?

Reply:
Yeah, though it would be great to also update the stub

Reply:
Looks like SimpleXmlElement now implements RecursiveIterator in PHP 8. I think need to fix how stubs are loaded for target php version first.

https://3v4l.org/bNNkh#v8.0.28
https://3v4l.org/bNNkh#v7.4.33